starter-code/pizza.js

```diff
@@ -81,14 +81,43 @@ function renderGlutenFreeCrust() {
 
 function renderButtons() {
   // Iteration 3: add/remove the class "active" of each `<button class="btn">`
+    const entries = Object.entries(state)
```

Comment:
Aca te esta saliendo activar y desactivar los ingredientes por que es lo unico que le estas pasando adentro de los switches, al final tu query selector nunca se esta corriendo.
Un approach mas sencillo para este problema seria un query selector para la clase .tb, luego hacer un for each que remueva la clase active. Asi setiamos de entrada los botones.

 ` document.querySelectorAll('.btn').forEach(function ($btn){`
  `  $btn.classList.remove('active')`
 ` })`

luego poner condiciones individuales por cada ingrediente: 
en este caso como el state de peperoni es true solo hay que poner state.peperone en la condicional.
Si es true, agrega la clase active.
 ` if( state.pepperonni ) document.querySelector('.btn.btn-pepperonni').classList.add('active')`